### DISCUSSION TITLE
Controlling dVRK Manipulators with rqt_gui

### DISCUSSION THREAD

**Discussion by zhengdae** [WPI-AIM/ambf#174] [Q&A]
Hey,

I am following the steps in "Controlling via RQT GUI".
I selected the .perspective file but nothing showed up.

Can you help me to understand what is going wrong?

**Comment by adnanmunawar** [comment]
Hello,
Thanks for the question and for pointing out the documentation issue. The perspective files were generated a while ago and we have since changed the models, (joint names, link names, passive joints, etc). As a result, those perspective files would not be valid anymore. You can still create your own publishers and plots using RQT and save the configuration as perspective files as it should be fairly straightforward. Alternatively, you can use more modern tools such as [Plotjuggler](http://wiki.ros.org/plotjuggler) for visualizing the the joint positions, for example. 

We shall update the documentation on the Wiki to reflect this answer.
